messages/password.generate.json

```diff
@@ -8,7 +8,11 @@
   "flags": {
     "onBehalfOf": "comma-separated list of usernames or aliases to assign the password to"
   },
-  "noSelfSetError": "Create a scratch org with the enableSetPasswordInApi org security setting set to TRUE and try again.",
+  "noSelfSetErrorV50": "Create a scratch org with the enableSetPasswordInApi org security setting set to TRUE and try again.",
+
+  "noSelfSetError": "Beginning in version 51, enableSetPasswordInApi is a feature in your scratch org definition file.  Settings.securitySettings.passwordPolicies.enableSetPasswordInApi is deprecated.",
```

Comment:
The following edit is a mouthful, but I tried to make it a bit more clear.  let me know if it's still correct!

---start edit---

Starting in Spring '21, EnableSetPasswordInApi is a feature in your scratch org definition file and not a setting. This change is a result of the Metadata API field Settings.securitySettings.passwordPolicies.enableSetPasswordInApi being deprecated in the Spring '21 release.

Reply:
perfect. using that verbatim.